charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml

```diff
@@ -45,18 +60,34 @@ spec:
       labels:
         app: helm-repository  # pods have this label, and Service and Deployment depend on it
```

Comment:
nit: if we are renaming to `helmChartsImage`, lets change labels and file name of the template